w3c.json.html

```diff
@@ -34,26 +34,70 @@ <h1>The <code>w3c.json</code> file</h1>
     {
         "group":      40318
     ,   "contacts":   ["darobin", "sideshowbarker"]
+    ,   "repo-type":  ["rec-track"]
     }
   </pre>
   <p>
     The fields that are understood at this point are:
   </p>
   <dl>
-    <dt><code>group</code></dt>
+    <dt id="group"><code>group</code></dt>
     <dd>
-      The numeric ID of the group in charge of this repo. While it is inconvenient for humans to
-      find such IDs they are the best thing we have to programmatically create links between
-      repositories and a whole lot of other data we have that are using this type of identifier.
+      The numeric ID of the group in charge of this repo. While it is
+      inconvenient for humans to find such IDs they are the best thing
+      we have to programmatically create links between repositories
+      and a whole lot of other data we have that are using this type
+      of identifier.
+      <br/><br/>
+      <strong>Note:</strong> If the group is actually a joint task
+      force of more than group, please specify all the IDs of the
+      groups consist of the task force as an array, e.g.:<br/>
+      &nbsp;&nbsp;&nbsp;&nbsp;"group": ["35422", "83907"]
```

Comment:
* Wrap this line of JSON in a `<pre>`, as in the example above.
* If we expect numeric IDs, and the example above uses a `number` type (not a `string`), these IDs here should be `number`, too (ie, remove the quotation marks).

Reply:
OK

I'll wrap the example line with "<pre>".

Also I'm fine with using "number" (=removing the quotation marks).